### ISSUE TITLE
Responders no longer work with custom block to create!

### ISSUE THREAD

**Issue by ineu** [activeadmin/inherited_resources#213]
I have such code in my create action:

```
create! {|format| format.html {redirect_to [:edit, @payment.account]}}
```

and expect it to use flash messages from Responders. It used to work before upgrading to Rails 3.2.2.
According to [this SO question](http://stackoverflow.com/questions/9530088/using-responders-with-a-respond-with-that-takes-a-block) respond_with behavior was changed. Recommendation from this question to create custom responder is useless as I want to use inherited_resources to write less code, not more. Is there any workaround for this issue?

**Comment by josevalim** [comment]
It is a rails bug, should be fixed on 3.2.3.

**Comment by ineu** [comment]
Thank you! Looking forward for 3.2.3.
